edge-hub/watchdog/linux/sh/blocking-operation.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# The most simple script i could come up with
+# for simulating a blocking thing - could be
+# `nginx -g 'daemon off;' or anything like that.
+for i in `seq 1 1000`; do
+  echo "$i\t `date +%s`"
+  sleep 1
+done
```

Comment:
This is just for testing. I can delete this later if we think we won't need it again.